common/src/main/java/org/apache/sedona/common/Predicates.java

```diff
@@ -13,9 +13,11 @@
  */
```

Comment:
Please put this one in `Functions`. `Predicates` are for functions that take two geometries as input and return true/false.

Reply:
@prantogg Can you address this?

Reply:
Done